docs/services/ses/README.md

```
@@ -0,0 +1,390 @@
+);
+
+await ses.sendEmail(message);
+
+// 1
+console.log(ses.sentEmails().length);
+```
+
+A rejection names every identity that failed the check in one message, the way real SES does, so a
+caller finds out everything it has to verify from one failure rather than several:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)